code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -34,6 +35,11 @@
 	icon_state = "bin_cypherkey"
 	translate_binary = TRUE
 
+/obj/item/encryptionkey/amplification
+	name = "amplification module key"
+	desc = "An amplification module key for a radio headset. It will allow the key applied radioset talk loudly."
```

Comment:
```suggestion
	desc = "An amplification module key for a radio headset. It will enable the "Loud mode" ability on any headset it is inserted into."
```

Reply:
No, these needs `\"`